kodi_addon_checker/check_addon_branches.py

```diff
@@ -60,14 +63,20 @@ def _get_addon_name(xml_path: str):
     return (tree.get("id"), tree.get("version"))
 
 
-def _is_pythonabi_compatible(target_branch, upper_branch):
+def _is_xbmcabi_compatible(dependencies, target_branch, upper_branch):
```

Comment:
Would prefer these to be typed, but I'm not really a python pro :)

Reply:
will add